source/clear-linux/tutorials/kubernetes.rst

```diff
@@ -239,8 +240,8 @@ configuration files for each package.
 
 If you customize any of the default package configuration files, you **must**
```

Comment:
Per the work-in-progress BKMs, we are using italic to emphasise text. So just a single *.